extension/src/Logging/ConsoleLogger.ts

```diff
@@ -12,9 +12,17 @@ export class ConsoleLogger implements ILogger {
   }
 
   log(message?: string, ...optionalParams: string[]): void {
```

Comment:
I wonder if we should change `optionalParams[]` to `unknown[]` to avoid errors like these:

![image](https://user-images.githubusercontent.com/17023248/150034052-27d26c0e-137e-43dc-9ea7-dab64c4f47d4.png)

(If you're not seeing these I'd love to know what setting I'm missing :sweat_smile: )

Reply:
Nope, I do not have that error... And I just commented out all my settings.json, and still do not get that error... 😕

So It seems to not be some settings I have, but some setting I am missing... 😜